news/126.bugfix

```diff
@@ -0,0 +1 @@
+Add documentation for `threads` option.
```

Comment:
this file should be named ``126.doc``, see https://towncrier.readthedocs.io/en/actual-freaking-docs/quickstart.html#creating-news-fragments

Reply:
done in https://github.com/plone/plone.recipe.zope2instance/pull/126/commits/8b387a95cef792da96a80c3d154b15dff687338e